biothings/web/applications.py

```diff
@@ -32,6 +32,8 @@
 from pydoc import locate
 from types import SimpleNamespace
 
+import sentry_sdk
+from sentry_sdk.integrations.tornado import TornadoIntegration
```

Comment:
Should make `sentry_sdk` optional (e.g. in the dev setup), just like how we import `raven` package below.

You can completely replace `raven` with `sentry_sdk`.